__tests__/api.test.js

```diff
@@ -315,3 +315,41 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles (topic query)", () => {
+  test("GET:200 responds with articles with specific topic", () => {
+    return request(app)
+      .get("/api/articles/topic/mitch")
+      .expect(200)
+      .then((res) => {
+        const articles = res.body.articles;
+        expect(articles.length).toBe(12);
+      });
+  });
```

Comment:
We could add an extra test to check what happens when a topic exists but has no article, like for topic=paper in our data 🙂